src/commandHandlers/help.ts

```diff
@@ -19,13 +19,15 @@ export const command = async (arg: [string, string], embed: MessageEmbed) => {
       false
     );
   } else {
-    commandList.forEach((commandItem) =>
-      embed.addField(
-        `${COMMAND_PREFIX}${commandItem.triggers[0]}`,
-        `${commandItem.description}\nUsage: ${COMMAND_PREFIX}${commandItem.usage}`,
-        false
-      )
-    );
+    let text = '';
+    commandList.forEach((commandItem) => {
+      text = text.concat(`**${COMMAND_PREFIX}${commandItem.triggers[0]}**  `);
+      text = text.concat(`${commandItem.description}\n`);
+      text = text.concat(
+        'Usage:' + '`' + `${COMMAND_PREFIX}${commandItem.usage}\n` + '`'
+      );
+    });
+    embed.addField(`\u200b`, `${text}`);
```

Comment:
Can just be `text`

```diff
- `${text}`
+ text
```

Reply:
I think this could change too?

```diff
- `\u200b`
+ '\u200b'
```

Reply:
oh yes.
Will do(I'm not well-versed in js/ts)

Reply:
No worries, we are all always learning - thats why I love getting my PRs reviewed, because people give me so many tips 🤓